code/style.css

```diff
@@ -84,68 +156,151 @@ select {
   display: none;
   flex-direction: column;
   align-items: center;
-  justify-content: center;
-  color: white;
-  text-align: center;
-  height: inherit;
+  color: var(--primary);
+  text-transform: uppercase;
+  cursor: pointer;
 }
 
-.card:hover {
-  background-color: var(--secondary);
+/****** CARD ENDS ******/
+
+/****** for those devices where hover doesn't work ******/
+
+@media (hover: hover) and (pointer: fine) {
+ 
+ .card:hover img,
+.card:hover p
```

Comment:
Really fun that you added the hover effects. Maybe the names are already revealed on the board at the start of the game, and the guess hover effect still comes in when you mouse over? Just a thought that could fill out the negative space!